pxtlib/sprite-editor/canvasGrid.ts

```diff
@@ -35,6 +38,23 @@ namespace pxtsprite {
             this.hideOverlay();
         }
 
+        setEyedropperMouse(on: boolean) {
+            const eyedropperClass = "sprite-editor-eyedropper";
+            if (on) {
+                this.paintLayer.classList.add(eyedropperClass);
+                if (!this.lightMode) {
+                    this.backgroundLayer.classList.add(eyedropperClass);
```

Comment:
Can't use classList (not supported by IE). I think we have a helper for this in one of the utils

Reply:
It looks like ie10+ supports the simple add/remove cases https://caniuse.com/#feat=classlist (it is also used over in https://github.com/microsoft/pxt/pull/5515#discussion_r279769731 and a handful of other parts of the codebase, so if it is broken we'd probably want to fix those too?)

I see a decent number of different helpers for handling it like you mentioned though, each with slightly different implementations:

* pxtblocks/fields/field_utils.ts
* pxtlib/svgUtil.ts
* pxt/pxtsim/instructions.ts
* pxt/pxtsim/runtime.ts
* pxtsim/svg.ts
* webapp/src/sui.tsx